ember-cli-build.js

```diff
@@ -3,16 +3,23 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  let app = new EmberAddon(defaults, {
-    // Add options here
-  });
+    let app = new EmberAddon(defaults, {
+        'ember-font-awesome': {
+            includeComponent: true
+        },
 
-  /*
+        'ember-bootstrap': {
+            bootstrapVersion: 4,
+            importBootstrapCSS: false
+        }
```

Comment:
is this bootstrap stuff just for the dummy app, or is that required for the addon to build?

Reply:
It's just for the dummy app